lab_support/get_a_book/BookSource.md

```diff
@@ -0,0 +1,5 @@
+# Don Quixote by Miguel de Cervantes Saavedra
+
+The provided text at [./book.txt](./book.txt) was downloaded from the [Gutenberg Project](http://www.gutenberg.org/wiki/Main_Page).
+
+This is the link where the book is available: [http://www.gutenberg.org/ebooks/996](http://www.gutenberg.org/ebooks/996)
```

Comment:
Delete as it has been merged with the previous bullet point

Reply:
Done